../../base/strings/string_util_impl_helpers.h:235:16: error: no member named 'iota' in namespace 'std::ranges'

--- a/base/strings/string_util_impl_helpers.h
+++ b/base/strings/string_util_impl_helpers.h
@@ -232,7 +232,7 @@ inline bool DoIsStringUTF8(std::string_v
 // Lookup table for fast ASCII case-insensitive comparison.
 inline constexpr std::array<unsigned char, 256> kToLower = []() {
   std::array<unsigned char, 256> table;
-  std::ranges::iota(table, 0);
+  std::iota(table.begin(), table.end(), 0);
   std::iota(table.begin() + size_t{'A'}, table.begin() + size_t{'Z'} + 1, 'a');
   return table;
 }();
